examples/hello/bpWriteReadHip/bpWriteReadHip.cpp

```diff
@@ -120,7 +120,7 @@ int main(int argc, char **argv)
         std::cout << "[BPWrite] error: " << hipGetErrorString(hipExit) << std::endl;
         return 1;
     }
-    const std::vector<std::string> list_of_engines = {"BP4", "BP5"};
+    const std::vector<std::string> list_of_engines = {"BPFile"};
```

Comment:
I'm wondering if this should be left as it was. It appears the original author may have wanted this to test out for both BP4 and BP5 since they loop through `list_of_engines`.

Reply:
I discussed that with @eisenhauer and he said that we should just remove it.